src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -54,45 +54,36 @@ public static float GetTotalNominalCapacity(IEnumerable<OrganelleTemplate> organ
     public static Dictionary<Compound, float> GetTotalSpecificCapacity(IReadOnlyList<OrganelleTemplate> organelles,
         out float nominalCapacity)
     {
-        var totalNominalCap = 0.0f;
+        var totalNominalCap = GetTotalNominalCapacity(organelles);
+        nominalCapacity = totalNominalCap;
 
-        int count = organelles.Count;
+        var capacities = new Dictionary<Compound, float>();
 
-        for (int i = 0; i < count; ++i)
-        {
-            var organelle = organelles[i];
-            totalNominalCap += GetNominalCapacityForOrganelle(organelle.Definition, organelle.Upgrades);
-        }
+        AddSpecificCapacity(organelles, capacities, totalNominalCap);
 
-        var capacities = new Dictionary<Compound, float>();
+        return capacities;
+    }
 
-        // Update the variant of this logic in UpdateSpecificCapacities if changes are made
-        for (int i = 0; i < count; ++i)
-        {
-            var organelle = organelles[i];
+    public static Dictionary<Compound, float> GetTotalSpecificCapacity(IEnumerable<CellTemplate> cells,
```

Comment:
I think this is different enough in purpose / stage this belongs in that I'd move this to a new file in `src/multicellular_stage/CellBodyPlanInternalCalculations.cs`